app/main.cpp

```diff
@@ -343,9 +343,15 @@ int main( int argc, char *argv[] )
 #else
   projDir = dataDir;
 #endif
-
+  AppSettings as;
   InputProjUtils inputProjUtils;
-  copy_demo_projects( demoDir, projectDir );
+
+  // copy demo projects if the app is launched for the first time
+  if ( !as.isAppInitialized() )
```

Comment:
can we use e.g. "demoProjectsCopied" instead of "appInitialized" ? The latter is very generic and it could mean anything - the former would be more descriptive IMHO :-)